index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE>
+<html>
+	<body>
+		<head> Welcome to the Land of Whatever you want it to be!<br>It's nice to meet you!</head>
+		<form action="javascript:onButtonClick()"> 
```

Comment:
Though this is technically correct, I was expecting an event handler to be used (given that we discussed that pattern at great length during the lectures).